model/textparse/openmetricsparse.go

```diff
@@ -94,16 +94,51 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+
+	// lines ending with _created are skipped by default as they are
+	// parsed by the CreatedTimestamp method. The skipCT flag is set to
+	// false when the CreatedTimestamp method is called.
+	skipCT bool
+}
+type openMetricsParserOptions struct {
+	// SkipCT skips the parsing of _created lines by default.
+	SkipCT bool
+}
+
+type OpenMetricsOption func(*openMetricsParserOptions)
+
+func WithSkipCT(skipCT bool) OpenMetricsOption {
```

Comment:
Nice. We have to change this name to prefix with OM parser, otherwise it feels like option for any parser, which is not. We can skip argument for bool options too.

```suggestion
func WithOpenMetricsParserCTSeriesSkipped() OpenMetricsOption {
```

Reply:
or `WithOMParserCTSeriesSkipped`

Reply:
applied this